lib/portage/dbapi/bintree.py

```diff
@@ -2049,7 +2036,17 @@ def dep_bestmatch(self, mydep):
             return ""
         return mymatch
 
-    def getname(self, cpv, allocate_new=None):
+    def getname(self, cpv, allocate_new=None, remote_binpkg_format=None):
+        return self._getname(
+            cpv, allocate_new=allocate_new, remote_binpkg_format=remote_binpkg_format
+        )[0]
+
+    def getname_build_id(self, cpv, allocate_new=None, remote_binpkg_format=None):
+        return self._getname(
+            cpv, allocate_new=allocate_new, remote_binpkg_format=remote_binpkg_format
+        )
+
+    def _getname(self, cpv, allocate_new=None, remote_binpkg_format=None):
```

Comment:
Any reason for the indirection though? It seems that `_getname` and `getname_build_id` are pretty much the same thing.

```suggestion
    def getname(self, cpv, allocate_new=None, remote_binpkg_format=None):
        return self.getname_build_id(
            cpv, allocate_new=allocate_new, remote_binpkg_format=remote_binpkg_format
        )[0]

    def getname_build_id(self, cpv, allocate_new=None, remote_binpkg_format=None):
```

Reply:
Oh, that's right.